lib/players/human.rb

```diff
@@ -0,0 +1,24 @@
+module Players
+  class Human < Player
+
+    attr_accessor:input
+
+    def user_input
```

Comment:
I would move these two method to lives inside of you CLI or GAME class. It seems out of place here.